doc/stdenv/stdenv.chapter.md

```diff
@@ -646,7 +646,7 @@ See the [build phase](#var-stdenv-makeFlags) for details.
 
 ##### `checkTarget` {#var-stdenv-checkTarget}
 
-The make target that runs the tests. Defaults to `check`.
+The make target that runs the tests. Defaults to `check` if it exists, otherwise `test`; if neither is found, do nothing.
```

Comment:
```suggestion
The Make target that runs the tests.
By default, use `check` if it exists, otherwise `test`; if neither is found, do nothing.
```

Capitalising the software project name helps distinguish it from the verb. I'm also fine with using `make`. The second sentence confused me, because it switched subjects after the semicolon. The variable *defaults* to something, but whatever uses it *does* something.

It would be worthwhile to note what happens if the target one specifies explicitly does not exist.